BrainPortal/app/views/bourreaux/_runtime_info.html.erb

```diff
@@ -58,7 +58,7 @@
       <% t.cell("Cluster Management System Type")     { red_if(@bourreau.cms_class != @info.bourreau_cms, @info.bourreau_cms) } %>
 
       <% t.cell("Workers Last Change Revision")       { @info.worker_lc_rev } %>
-      <% t.cell("Cluster Management System Revision") { @info.bourreau_cms_rev.svn_id_pretty_rev_author_date } %>
+      <% t.cell("Cluster Management System Revision") { @info.bourreau_cms_rev} %>
```

Comment:
To make it more like the original behavior, I suggest you modify the code in bourreau.rb so that the value of bourreau_cms_rev has just the "commit_author_date" instead of what .to_s() creates right now. This is line 250 of bourreau.rb, which is currently unchanged (and I can't comment directly on it in the pull request).